### ISSUE TITLE
[🐞]: all_stones item looks like barrier

### ISSUE THREAD

**Issue by VoidLeech** [MehVahdJukaar/StoneZone#14]
### Code of Conduct

- [X] Always check you are using the latest version of the mods and its dependencies
- [X] Ensure the version of EveryCompat & Moonlight Lib are up to date
- [X] Remove mod that enhances Minecraft: Optifine, Sodium, others. The issue still persists.
- [X] If you are unsure which mod is the culprit.
Disable all of your mods and enable them 1-2 mods each time to isolate the culprit
- [X] Confirm that there is no existing issue with a similar description submitted in the list of issues.

### Version - Loader (DO NOT FORGET TO SELECT THE CORRECT VERSION)

1.20.1 - FORGE

### StoneZone Version

stonezone-1.20-2.7.1

### Moonlight Lib Version

moonlight-1.20-2.13.41

### Issue with mods

Any furniture mod (I used Create)
Any stone mod (I used a couple)

### Issue Detail

![image](https://github.com/user-attachments/assets/3200057d-212b-432e-ba21-e12a898f864c)
The all_stones item, which I assume would cycle through all stones, instead looks like a barrier.

**Comment by MehVahdJukaar** [comment]
It's a bug . The mod is supposed to crash here

**Comment by MehVahdJukaar** [comment]
As that can only be barrier if the mod did not generate any block

**Comment by VoidLeech** [comment]
This screenshot is from a session where stonezone did generate blocks...

**Comment by MehVahdJukaar** [comment]
Uhmmm and items too?
Maybe last update broke the item renderer

**Comment by VoidLeech** [comment]
Yes. I'll go get the debug.log to see if there's anything in there (there wasn't in latest.log that I saw)

**Comment by VoidLeech** [comment]
![image](https://github.com/user-attachments/assets/a86ffe41-154e-4121-8bf0-f28d54b149ae)
https://mclo.gs/YhVzZP2

**Comment by Xelbayria** [comment]
Try v2.7.2 and let me know if the issue is resolved. (it will take a while for the version to get approval on Curseforge)

**Comment by VoidLeech** [comment]
Uhh okay update, it seems that when using a different furniture mod (bbb), the item does cycle through the added items from that mod, but still not items added from create.

**Comment by Xelbayria** [comment]
> Uhh okay update, it seems that when using a different furniture mod (bbb), the item does cycle through the added items from that mod, but still not items added from create.

Yea I noticed this. It's so strange that Create's is not included.

**Comment by Xelbayria** [comment]
Will closing this because I am not sure how to fix it. You just need to add another mod so that should fix the issue. 

idk why create's not icnluded.
